chef/cookbooks/metasploitable/recipes/elasticsearch.rb

```diff
@@ -0,0 +1,73 @@
+#
+# Cookbook:: metasploitable
+# Recipe:: elasticsearch
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+include_recipe 'metasploitable::7zip'
+include_recipe 'metasploitable::jdk8'
+
+remote_file 'C:\Windows\Temp\elasticsearch-1.1.1.zip' do
+  source 'http://repo1.maven.org/maven2/org/elasticsearch/elasticsearch/1.1.1/elasticsearch-1.1.1.zip'
+  action :create
+end
+
+execute 'Extracting files' do
+  command '"C:\Program Files\7-Zip\7z.exe" x "C:\Windows\Temp\elasticsearch-1.1.1.zip" -o"C:\Program Files\"'
+  action :run
+end
+
+execute 'Install ElasticSearch' do
+  command '"C:\Program Files\elasticsearch-1.1.1\bin\service.bat" install'
+  environment ({"JAVA_HOME" => "C:\\Program Files\\Java\\jdk1.8.0_144"})
+  only_if { ::File.exist?('C:\Program Files\Java\jdk1.8.0_144\bin\java.exe') }
+  action :run
+end
+
+windows_service 'elasticsearch-service-x64' do
+  action :enable
+  startup_type :automatic
+end
+
+execute 'Starting service' do
+  command '"C:\Program Files\elasticsearch-1.1.1\bin\service.bat" start'
```

Comment:
Why not using actions `[:enable, :start]` for `windows_service[elasticsearch-service-x64]` resource? instead of this `execute`?